app/Http/Controllers/Api/V1/CohortRequestController.php

```diff
@@ -1363,4 +1367,246 @@ public function byUser(Request $request, int $id): JsonResponse
             throw new Exception($e->getMessage());
         }
     }
+
+    /* @OA\Post(
+        *    path="/api/v1/cohort_requests/user/{id}/request_nhse_access",
+        *    operationId="post_nhse_access_cohort_request_by_user",
+        *    tags={"Cohort Requests"},
+        *    summary="CohortRequestController@requestNhseAccess",
+        *    description="Indicates the user has begun the NHSE SDE access process by contacting NHS via the Gateway button",
+        *    security={{"bearerAuth":{}}},
+        *    @OA\Parameter(
+        *       name="id",
+        *       in="path",
+        *       description="user id",
+        *       required=true,
+        *       example="1",
+        *       @OA\Schema(
+        *          type="integer",
+        *          description="user id",
+        *       ),
+        *    ),
+        *    @OA\Response(
+        *       response="200",
+        *       description="Success response",
+        *       @OA\JsonContent(
+        *         @OA\Items(type="object",
+        *           @OA\Property(property="id", type="integer", example="123"),
+        *           @OA\Property(property="user_id", type="integer", example="1"),
+        *           @OA\Property(property="request_status", type="string", example="PENDING"),
+        *           @OA\Property(property="cohort_status", type="boolean", example="0"),
+        *           @OA\Property(property="request_expire_at", type="datetime", example="2023-04-03 12:00:00"),
+        *           @OA\Property(property="created_at", type="datetime", example="2023-04-03 12:00:00"),
+        *           @OA\Property(property="updated_at", type="datetime", example="2023-04-03 12:00:00"),
+        *           @OA\Property(property="deleted_at", type="datetime", example="2023-04-03 12:00:00"),
+        *           @OA\Property(property="accept_declaration", type="boolean", example="0"),
+        *         ),
+        *       ),
+        *    ),
+        * )
+        */
+    public function requestNhseAccess(Request $request, int $id): JsonResponse
+    {
+        $input = $request->all();
+        $jwtUser = array_key_exists('jwt_user', $input) ? $input['jwt_user'] : [];
+
+        // Check that the user is asking only for their own record.
+        if (!($jwtUser['id'] === $id)) {
+            throw new UnauthorizedException();
+        }
+
+        try {
+            $cohortRequest = CohortRequest::where('user_id', (int)$id)->first();
```

Comment:
Just a note, you shouldn't _have_ to cast this `$id` as it's injected via dependency under strict type. Not a massive issue, just a note.